.devcontainer/Dockerfile

```diff
@@ -8,7 +8,7 @@ FROM docker.io/library/debian:11
 RUN \
   apt-get update \
   && apt-get install -y curl \
-  && curl -sL https://deb.nodesource.com/setup_14.x | bash - \
+  && curl -sL https://deb.nodesource.com/setup_16.x | bash - \
```

Comment:
changes look fine, just want to confirm youve tested this out in a new codespace?

Reply:
Sorry, not sure what you mean by new codespace? I did run the tests in core repo (integration and unit) for all projects from within the rebuilt container using this newer Dockerfile

Reply:
ok thats good enough;

GitHub allows you to spin up a [Codespace](https://github.com/features/codespaces) using these devcontainers; didnt realize its now a paid service, but used to be how id test things online